sleap_nn/data/providers.py

```
@@ -28,6 +28,71 @@
+        max_instances: Maximum number of instances that could occur in a single LabeledFrame.
+        user_instances_only: True if filter labels only to user instances else False.
+            Default: True.
+
+    Returns:
+        Dict with image, instancs, frame index, video index, original image size and
+        number of instances.
+
+    """
+    # Filter to user instances
+    if user_instances_only:
+        if lf.user_instances is not None and len(lf.user_instances) > 0:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)